src/views/catalog/templatescatalog/components/TemplatesCatalogTile.tsx

```diff
@@ -47,10 +53,13 @@ export const TemplateTile: React.FC<TemplateTileProps> = React.memo(
     return (
       <div onClick={() => onClick(template)}>
         <CatalogTile
-          badges={
+          badges={[
             bootSourcesLoaded
               ? isBootSourceAvailable && [
-                  <Badge key="available-boot">{t('Source available')}</Badge>,
+                  <Stack className="badge-stack" key="badge-stack">
+                    <Badge key="available-boot">{t('Source available')}</Badge>
+                    {isDeprecated ? <DeprecatedBadge className="deprecated-template" /> : null}
```

Comment:
With that the deprecated badge will be added only if the source is available. I think you should add the badge to the array only if the template is deprecated

Reply:
Good catch, thanks! Fixed!